v1/lib/baseOutput.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: Max
+ * Date: 05.02.2016
+ * Time: 09:50
+ */
+abstract class BaseOutput
+{
+    abstract public function output(GameFieldController $_gameFieldController, $_cycle);
```

Comment:
Sorry, but this is just to less. You need to start thinking on how it is for another developer, if he needs to implement a new Output plugin. Describe here, what he needs to implement there.

See thhis screenshot on how this can be done ;-)
![abstractfunctiondocumentation](https://cloud.githubusercontent.com/assets/965802/12843742/9c7ebd32-cbfb-11e5-9fa9-632ea17ba3f7.PNG)